src/lib/containers/widgets/ImageFileHandle.tsx

```diff
@@ -0,0 +1,64 @@
+import { FileHandleAssociateType } from '../../utils/synapseTypes'
```

Comment:
Very similar to FileHandleLink, but even less complicated (no display text value or download icon)